compiler/next/lib/frontend/Parser/chapel.ypp

```diff
@@ -99,15 +99,6 @@
     ThrowsTag_THROWS,
   };
 
-  enum ShadowTag {
-    ShadowTag_CONST,
-    ShadowTag_IN,
-    ShadowTag_CONST_IN,
-    ShadowTag_REF,
-    ShadowTag_CONST_REF,
-    ShadowTag_VAR,
-  };
-
```

Comment:
I think ReturnTag and IntentTag below do not need to be here - use Function's and Formal's enums instead?